src/Test/IntegrationCase.php

```diff
@@ -128,13 +128,17 @@ public function getRequirement($name)
         if (!is_string($name)) {
             throw new \InvalidArgumentException(sprintf(
                 'Requirement key must be a string, got "%s".',
-                is_object($name) ? get_class($name) : gettype($name).'#'.$name));
+                is_object($name) ? get_class($name) : gettype($name).'#'.$name
+            ));
         }
 
         if (!array_key_exists($name, $this->requirements)) {
-            throw new \InvalidArgumentException(sprintf(
+            throw new \InvalidArgumentException(
+                sprintf(
                 'Unknown requirement key "%s", expected any of "%s".',
-                $name, implode('","', array_keys($this->requirements)))
+                $name,
+                implode('","', array_keys($this->requirements))
+            )
```

Comment:
The `sprintf(` part should remain on the same line as before so that indentation stays valid. Then both closing parentheses would be on the same line too. Same for similar changes in other files.